digdag-docs/src/operators/require.md

```diff
@@ -71,3 +88,8 @@
   ignore_failure: true
   ```
 
+  require> evaluates *ignore_failure* at last of its process. If rerun_on is set and require> run new attempt, the result of new attempt is checked.
+
+## Notes
+- require> has been changed to ignore inherited *retry_attempt_name* parameter. 
```

Comment:
Adding `since v0.10` or something may be helpful for users, I think.

Reply:
Good point. But I am planning back port to 0.9.xx.